src/main/java/com/conveyal/analysis/controllers/RegionalAnalysisController.java

```diff
@@ -550,10 +674,12 @@ public void registerEndpoints (spark.Service sparkService) {
         });
         sparkService.path("/api/regional", () -> {
             // For grids, no transformer is supplied: render raw bytes or input stream rather than transforming to JSON.
+            // Wait, do we have any endpoints that write grids into responses? It looks like they all return URLs now.
```

Comment:
See `TimeGridWriter.writeGeotiff`?

Reply:
But now I'm wondering what causes the RegionalAnalysis object returned by getRegionalAnalysis to be serialized properly.

Reply:
We looked at this today, and RegionalAnalysis inherits a toString method that produces JSON. Leaving as-is.